src/components/MatPassContent.tsx

```
@@ -1065,20 +1338,64 @@ export default function MatPassContent() {
+                   transition-all duration-300 transform
+                   disabled:opacity-50 disabled:cursor-not-allowed
+                   bg-gradient-to-r from-green-500 to-teal-500 
+                   hover:from-green-600 hover:to-teal-600
+                   active:scale-95 hover:shadow-lg
+                   flex items-center justify-center gap-2
+                   relative overflow-hidden group"
+            >
+              {/* Hover shine effect */}
+              <div className="absolute top-0 -inset-full h-full w-1/2 z-5 block transform -skew-x-12 bg-gradient-to-r from-transparent to-white opacity-20 group-hover:translate-x-full transition-transform duration-1000"></div>
+              
+              <div className="relative flex items-center justify-center gap-2">
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)